starlark/src/eval/mod.rs

```diff
@@ -499,9 +498,8 @@ fn eval_dot<'a>(
 enum TransformedExpr {
     Dot(Value, String, Span),
     ArrayIndirection(Value, Value, Span),
-    List(Vec<TransformedExpr>, Span),
-    Tuple(Vec<TransformedExpr>, Span),
-    OtherExpr(AstExpr),
+    Identifier(AstString),
+    Slot(usize, AstString),
```

Comment:
I haven't fully wrapped my head around that change, does that means we no longer used "TransformedExpr" for the normal assignment?

Reply:
We didn't use it at least for a long time. Assign in master is:

```
        Statement::Assign(ref lhs, AssignOp::Assign, ref rhs) => {
            let rhs = eval_expr(rhs, context)?;
            set_expr(lhs, context, rhs)
        }
```